vocabularies/Common.xml

```diff
@@ -74,6 +74,18 @@
         <Annotation Term="Core.IsLanguageDependent" />
       </Term>
 
+      <Term Name="DocumentationRef" Type="Edm.String" Nullable="false" AppliesTo="EntityType Property Parameter ReturnType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.IsURL" />
+        <Annotation Term="Core.Description" String="A URI referencing language-dependent documentation for the annotated model element" />
+        <Annotation Term="Core.Example">
+          <Record Type="Core.PrimitiveExampleValue">
+            <PropertyValue Property="Description" String="URN scheme understood by SAP Companion" />
```

Comment:
This product has in the last two or three years at least three product name changes - not sure if I would mention it here. Is there a difference to "normal" URLs?

Reply:
Yes. This is an example from one of our test services.
> `<Annotation Term="SAP__common.DocumentationRef" String="urn:sap-com:documentation:key?=type=DE&id=SPRAS"/>`

So it is in the end pretty ABAP specific. If we would want to enhance it for other backends, also Companion would need to learn about that.

Reply:
> Yes. This is an example from one of our test services.
> 
> > `<Annotation Term="SAP__common.DocumentationRef" String="urn:sap-com:documentation:key?=type=DE&id=SPRAS"/>`
> 
> So it is in the end pretty ABAP specific. If we would want to enhance it for other backends, also Companion would need to learn about that.

OK - thanks, @uhlmannm . The question is if this urn schema was also used for xray, for FLP In-App Help, and Web Assistant, which is now used for SAP Companion - so either it's only relevant for this specific tool, or it's relevant for the in-app help framework from SAP. Usually we want to define here a "timeless" abstraction ...

Reply:
This is just an _example_.

Reply:
Thanks, @HeikoTheissen!
Nevertheless, it's only yesterday that I heard that there is the next name under way. I assume that in two years nobody knows what companion is/was. But don't mind - it was just a comment. For me the term itself is completely fine.

Reply:
But in the end they evaluate the schema. Should we write rather "URN scheme containing the information to identify the help text"?